clang/test/CXX/cpp/cpp.module/p1.cppm

```diff
@@ -0,0 +1,13 @@
+// RUN: %clang_cc1 -std=c++20 -emit-module-interface %s -triple x86_64-linux-gnu -DTEST=1 -verify
+// RUN: %clang_cc1 -std=c++20 -emit-module-interface %s -triple x86_64-linux-gnu -DTEST=2 -verify
+
+module;
+export module x;
+#include "version.h"
+#if TEST == 1
```

Comment:
we prefer using split-file now.

Reply:
Is that documented somewhere?

Reply:
https://llvm.org/docs/TestingGuide.html#extra-files

Also it is somewhat clearly that split file is better than the sections guarded by #if-#endif. We can find the example by searching `split-file` under clang/test/Modules

Reply:
Agree!

Reply:
Done!